### PR TITLE
Tc1001

### PR DESCRIPTION
**Pull request by PatelSarita**
Local test completed, ready for review

Ticket : www.jira.com/TC1001

Local Test result;

### CODE REVIEW

File: src/LoginTest.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 public class LoginTest {
     public static void main(String[] args) {
         System.out.println("Local login Test!!!");
+        System.out.println("Local login Test edited");
+        System.out.println("Please review");
```
Comment (RosanaKmetovikj) [inline src/LoginTest.java:5]: Can you remove line 5
create new test cases for user name and password

**Comment by RosanaKmetovikj** [comment]
Sarita pleace remove line 5 in LoginTest

File: src/LoginTest.java (hunk 1 above)
Comment (RosanaKmetovikj) [reply src/LoginTest.java:5]: Can you remove line 5 in LoginTest

**Comment by RosanaKmetovikj** [comment]
Can you delete line 5 from LoginTest

File: src/LoginTest.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
 public class LoginTest {
     public static void main(String[] args) {
         System.out.println("Local login Test!!!");
+        System.out.println("Local login Test edited");
```
Comment (RosanaKmetovikj) [inline src/LoginTest.java:4]: just edit this one

File: src/LoginTest.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+public class LoginTest {
+    public static void main(String[] args) {
+        System.out.println("Local login Test!!!");
```
Comment (RosanaKmetovikj) [inline src/LoginTest.java:3]: make changes here

**Review by RosanaKmetovikj — APPROVED** [review]

File: src/LoginTest.java (hunk 2 above)
Comment (PatelSarita) [reply src/LoginTest.java:4]: ok...done

**Pull request merged**